app/src/main/java/com/github/epfl/meili/util/ClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.github.epfl.meili.util
+
+interface ClickListener {
+    fun onClicked(buttonId: Int, info: String)
+}
```

Comment:
I'm not sure I understand the purpose of this interface, could you please explain? Also, some documentation is needed here.

Reply:
This is implemented by Activities so that other classes can call signal it that a button has been clicked. It should only be used by sub activities (mainly Recycler Views) to inform of such clicks in their view. Otherwise the activity would only know that one of the friends for example (from the Recycler View) has been clicked but it wouldn't know which one in concrete. I hope this was clear, and I am adding documentation in the next commit. Thanks!

Reply:
Ah, makes sense, thanks a lot for explaining!